default.json

```
@@ -79,7 +79,7 @@
       "matchManagers": ["npm"],
       "matchPackageNames": ["dd-trace"],
 
-      "allowedVersions": "!/^4.21.0$/"
+      "allowedVersions": "!/^(3.42.0|4.21.0)$/"
```

Nitpick, but should these be e.g.:
```suggestion
      "allowedVersions": "!/^(3\\.42\\.0|4\\.21\\.0)$/"
```

---

I did look at this and couldn't think of how you'd defeat the regex between it enforcing boundaries `^$` and npm using semver.

But this is more correct 👍